mycli/main.py

```diff
@@ -1086,14 +1106,17 @@ def get_last_query(self):
               help='Warn before running a destructive query.')
 @click.option('--local-infile', type=bool,
               help='Enable/disable LOAD DATA LOCAL INFILE.')
-@click.option('-g', '--login-path', type=str,
+@click.option('--login-path', type=str,
               help='Read this path from the login file.')
 @click.option('-e', '--execute',  type=str,
               help='Execute command and quit.')
 @click.option('--init-command', type=str,
               help='SQL statement to execute after connecting.')
 @click.option('--charset', type=str,
               help='Character set for MySQL session.')
+@click.option('--password-file', type=click.Path(),
+              help='File or FIFO path containing the password to connect to the db if not specified otherwise')
```

Comment:
CI fails because there is no period at the end of the help text

Reply:
fixed